job/client/src/main/java/alluxio/worker/job/JobMasterClient.java

```diff
@@ -15,7 +15,7 @@
 import alluxio.exception.AlluxioException;
 import alluxio.exception.ConnectionFailedException;
 import alluxio.grpc.JobCommand;
-import alluxio.grpc.TaskInfo;
+import alluxio.grpc.JobInfo;
```

Comment:
This class should be in the `server` module instead of `client`. Though I think its not something this refactor introduced. Although it is a `client`, we use the `client` module for client applications to the servers and we use the server modules to hold clients that servers use to communicate with each other.

Reply:
That is correct. This file doesn't have much of a business being in the client module.